storage/innobase/include/trx0vtq.h

```diff
@@ -0,0 +1,40 @@
+#ifndef trx0vtq_h
+#define trx0vtq_h
+
+/* Copyright (c) 2016, MariaDB Corporation.
```

Comment:
2016, 2017

Reply:
Not needed as it is to be merged into `trx0trx.h`. https://github.com/tempesta-tech/mariadb/issues/306